src/pipeline.jl

```diff
@@ -363,6 +363,7 @@ function RecipesPipeline.add_series!(plt::Plot, plotattributes)
             plotattributes[letter2], plotattributes[letter1]
     end
     _expand_subplot_extrema(sp, plotattributes, plotattributes[:seriestype])
+    plotattributes[:series_index] = 1
```

Comment:
This gets caught as an extra keyword argument

Reply:
Thanks, this was really just a placeholder to make things not break. What does that mean here, though? Would I just have to register it in `args.jl` somewhere? And at what point does it process that? (I ask in part because in another PR I previously added, I have a `:clims_calculated` key used internally, but I don't think I added it there.)